docs/source/tutorials_source/package/tutorial_moco_memory_bank.py

```diff
@@ -191,19 +193,32 @@
 # ---------------------------
 # Now we create our MoCo model. We use PyTorch Lightning to train
 # our model. We follow the specification of the lightning module.
-# In this example we set the number of feature for the hidden dimension to 128.
+# In this example we set the number of features for the hidden dimension to 512.
 # The momentum for the Momentum Encoder is set to 0.99 (default is 0.999) since
 # other reports show that this works better for Cifar-10.
+#
+# .. note:: We use a split batch norm to simulate multi-gpu behaviour. Combined
+#   with the use of batch shuffling, this prevents the model from communicating 
+#   through the batch norm layers.
 
 class MocoModel(pl.LightningModule):
     def __init__(self):
         super().__init__()
+        
+        # create a ResNet backbone and remove the classification head
+        resnet = lightly.models.ResNetGenerator('resnet-18', 1, num_splits=8)
+        backbone = nn.Sequential(
+            *list(resnet.children())[:-1],
+            nn.AdaptiveAvgPool2d(1),
+        )
+
```

Comment:
We will keep the `ResNetGenerator` for now to uphold backwards compatability but with the new implementation of `MoCo` and `SimCLR` (which are now independent from ResNet), it will be much easier to swap to the torchvision models. For now, I think it would be best if the new models don't depend on a specific model.